README.md

```diff
@@ -20,20 +20,47 @@ Only the latest nightly build is kept.
 Prerequisites
 -------------
 
-#### *nix building
+### *nix building
 
+###### Minimum required packages:
```

Comment:
It looks like this includes recommended packages in addition to "minimal required".

Reply:
I lazily added the additional audio driver options. They are now added to a new section.